common/client.go

```diff
@@ -334,17 +380,25 @@ func (rc *RestClient) Delete(url string, data interface{}, result interface{}) e
 	return err
 }
 
+// Put applies PUT method to the specified URL
+func (rc *RestClient) Put(url string, data interface{}, result interface{}) error {
+	err := rc.execMethod("PUT", url, data, result)
+	return err
+}
+
 // Get applies GET method to the specified URL,
 // putting the result into the provided interface
```

Comment:
You are mentioning here that the results are stored in the "provided interface". I think the same is true for POST, PUT and DELETE. You might want to mention it there as well in the docstrings.